sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/LiveMetricsExtensions.cs

```diff
@@ -63,9 +65,9 @@ public static class LiveMetricsExtensions
                     exporterOptions = sp.GetRequiredService<IOptionsMonitor<LiveMetricsExporterOptions>>().Get(finalOptionsName);
                 }
 
-                DoubleBuffer doubleBuffer = new();
-
-                return new LiveMetricsExtractionProcessor(doubleBuffer, new LiveMetricsExporter(doubleBuffer, exporterOptions));
+                // INITIALIZE INTERNALS
+                var manager = ManagerFactory.Instance.Get(exporterOptions);
```

Comment:
This should be cleaned up later. We will have one instance and do not need a factory.

Reply:
I have concerns about this. If users invoke `AddLiveMetrics` twice or use multiple ikeys.
I want to use the factory to guarantee that we only have once instance of LiveMetrics per ikey.